tests/unit/tunnels/SeleniumTunnel.ts

```diff
@@ -71,7 +71,7 @@ registerSuite('tunnels/SeleniumTunnel', {
       const oldLog = console.log;
 
       return {
-        afterEach() {
+        afterEachSeleniumTunnelTest() {
```

Comment:
Hah, the warning caught something!

Since we still want the `afterEach` to run, the fix here should actually be to move the tests into a `tests` object, like:

```js
return {
  afterEach() { ... },
  tests: {
    '3.0.0': createTest(...),
    '3.5.0': createTest(...),
    ...
  }
}
```

Reply:
Tried this code and I'm still seeing the afterEach warning message 🤔